_people/Suurbhi_Rathore

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+name: {Surbhi Rathore}
+role: {Graduate Research Assistant}
+importance: 7
+status: active
+---
+
+
+<!-- move any of the following you'd like to include into the yaml header
+github: surbhir08
```

Comment:
do you want the link to your github to display? if so copy this line into the header above the `---` on line 7